### DISCUSSION TITLE
all containers state is WAIT_START

### DISCUSSION THREAD

**Discussion by cyber7878** [Security-Onion-Solutions/securityonion#6805] [Unsupported Versions] [answered]
hey, i rebooted my system and when i check the state of the containers (with "so-status") all of then had a state of "WAIT_START"
when i try to run the command "sudo salt-call state.highstate", it gives me an error:

"rendering sls 'base:firewall' failed: jinja variable list object has no element 0
/var/cache/salt/minions/files/base/firewall/hostgroups.yaml (23)"

and then it gives me the same error with 'base:playbook':

"rendering sls 'base:playbook' failed: jinja variable list object has no element 0"

i tried to put line 23 in the hostgroups.yaml file under a comment ("#") but it still shows the same error.
do anyone have any idea what to do?

**Comment by m0duspwnens** [comment, marked as answer]
Hi @cyber7878. The issue seems similar to [https://github.com/Security-Onion-Solutions/securityonion/discussions/6798](https://github.com/Security-Onion-Solutions/securityonion/discussions/6798). If this is a manager, then check that the pillar values for `manager:mainint` and `host:mainint` match the interface name that is used to connect to the network and has an IP.

**Reply by cyber7878** [reply]
Thanks!! It worked 😄
